Makefile

```diff
@@ -193,6 +194,11 @@ aevm-test: aevm-test-deps
 	@./rebar3 eunit --application=aevm
 
 aevm-test-deps: $(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests
+aevm-test-deps:
+	$(eval VER=$(shell git -C "$(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests" log -1 --pretty=format:"%H"))
+	@if [ "$(VER)" != "$(AEVM_EXTERNAL_TEST_VERSION)" ]; then \
```

Comment:
I'm not sure why we'd need this explicit check, I guess git does that internal and much faster 🤔

Reply:
You are right. I just wanted to be a bit more silent if everything was ok.